UnityProject/Assets/Scripts/Furniture/BuckleInteract.cs

```diff
@@ -58,6 +58,9 @@ public void ServerPerformInteraction(MouseDrop drop)
 		var playerMove = drop.UsedObject.GetComponent<PlayerMove>();
 		playerMove.ServerBuckle(gameObject, OnUnbuckle);
 
+		var objectIntegrity = gameObject.GetComponent<Integrity>();
```

Comment:
do you think it's worth having an reference for it so don't have to get component all the time?